src/ui/components/Badge/styles.scss

```diff
@@ -10,16 +10,25 @@
   display: inline-block;
   font-size: $font-size-s;
   line-height: 1.3;
+  margin-top: 10px;
   padding: 8px 12px 6px;
   text-align: center;
+
+  .Icon {
+    @include margin-end(10px);
+
+    vertical-align: middle;
+  }
 }
 
 .Badge-featured {
   background-color: $teal-50;
+}
 
-  .Icon-featured {
-    @include margin-end(10px);
+.Badge-restart-required {
+  background-color: $yellow-50;
 
-    vertical-align: middle;
+  .Icon-restart-required {
+    margin-top: -2px;
```

Comment:
Well actually, `vertical-align` changes the other badge, so I'd like to keep this negative margin..